src/kit.ts

```diff
@@ -461,12 +455,7 @@ function kitTargetArch(hostTargetArch: string): string {
  * @param targetArch The architecture of the target
  */
 function kitHostTargetArch(hostArch: string, targetArch: string): string {
-  let hostTargetArch: string = hostArch;
-  if (targetArch && vsHostTargetArchsFromGenPlatform[targetArch] !== hostArch) {
-    hostTargetArch = hostTargetArch.concat("_").concat(vsHostTargetArchsFromGenPlatform[targetArch]);
-  }
-
-  return hostTargetArch;
+  return hostArch === targetArch ? hostArch : hostArch.concat("_").concat(targetArch);
```

Comment:
The second case is probably better as:
```ts
`${hostArch}_${targetArch}`
```
Using backtick quotes will evaluate the variables.